cockatrice/src/thememanager.cpp

```diff
@@ -107,3 +121,8 @@ void ThemeManager::themeChangedSlot()
 
     emit themeChanged();
 }
+
+QBrush ThemeManager::getExtraTableBgBrush(QString extraNumber, QBrush fallbackBrush)
+{
+    return loadExtraBrush(TABLEZONE_BG_NAME + extraNumber, fallbackBrush);
```

Comment:
it'd be preferable to store these in an array or map instead of needing to load them repeatedly.